planus-cli/src/intermediate_language/translation.rs

```diff
@@ -1114,28 +1143,12 @@ impl<'a> Translator<'a> {
 
     fn translate_enum(&self, current_file_id: FileId, decl: &ast::Enum) -> Enum {
         let mut alignment = decl.type_.byte_size();
-        for m in &decl.metadata {
+        for m in &decl.metadata.values {
```

Comment:
Check if force_align should work for enums

Reply:
It shouldn't. I have submitted google/flatbuffers#7515 upstream.